### PR TITLE
#166240965: Implement users should be able to view notifications

### PR DESCRIPTION
**Pull request by mifeille**
#### What does this PR do?
This PR will enable users to see their in-app notifications.

#### Description of Task to be completed?
Implement the task
Tests components, actions, and reducers

#### How should this be manually tested?
This PR can be tested by cloning this branch and running the development server with this command `yarn dev`

#### Screenshots (if appropriate)
ezgif com-video-to-gif (14)

#### What are the relevant pivotal tracker stories?
#166240965

### CODE REVIEW

File: src/utils/constants.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const BASE_URL = 'http://localhost:3000';
-export const BACKEND_URL = 'https://tesla-ah-staging.herokuapp.com';
-export const API_URL = 'https://tesla-ah-staging.herokuapp.com/api';
+export const API_URL = 'http://localhost:3000/api';
```
Comment (deschantkn) [inline src/utils/constants.js]: You can't use localhost.

Comment (mifeille) [reply src/utils/constants.js]: what do you mean?

Comment (deschantkn) [reply src/utils/constants.js]: The localhost url will not work when we deploy. Please update it.

File: src/components/layouts/Header.js [hunk 2]
```diff
@@ -10,6 +12,7 @@ import PropTypes from 'prop-types';
 import logo from '../../assets/img/logo_st.png';
 import { initProfile } from '../../redux/actions/profile.actions';
 import Profilemenu from '../menu/Profile';
+import { getUnreadNotifications, readNotification } from '../../redux/actions/notifications.actions';
 // import { setLoggedIn } from '../../redux/actions/ui.actions';
```
Comment (eliemugenzi) [inline src/components/layouts/Header.js]: Please delete this commented line

**Review by eliemugenzi — CHANGES_REQUESTED** [review]

File: src/components/layouts/Header.js (hunk 2 above)
Comment (mifeille) [reply src/components/layouts/Header.js]: it's not my line but I am going to delete it!

Comment (eliemugenzi) [reply src/components/layouts/Header.js]: Thank you @mifeille

**Review by deschantkn — APPROVED** [review]

**Review by EmyRukundo — APPROVED** [review]

**Review by eliemugenzi — APPROVED** [review]

**Pull request merged**